### ISSUE TITLE
Display space name in drawing area

### ISSUE THREAD

**Issue by ajwinter** [NREL/openstudio-geometry-editor#11]
Display the name of the space inside the space geometry in the drawing pane

**Comment by ljbrackney** [comment]
Labels are ginormous.

![labels](https://cloud.githubusercontent.com/assets/2230150/26644493/b268f2b6-45e9-11e7-8355-643497540019.png)

**Comment by shorowit** [comment]
Weird, I'm not seeing that.

**Comment by ljbrackney** [comment]
Rebuilt from 6/1 AM commits and retested.  Labels look good now.  Thanks!

**Comment by ljbrackney** [comment]
Ugh.  Spoke too soon.  I just switched to a different property and the problem is still there.  Very strange - one address and adding geometry looks fine.  A different one is the same as above.

**Comment by ljbrackney** [comment]
Good result - Commercial Building - La Kretz Innovation Campus in LA

![ok](https://cloud.githubusercontent.com/assets/2230150/26690621/d1a3034a-46ae-11e7-8d73-2801c4facbd6.png)

Bad result - Residential Building - Residence in Van Nuys

![notok](https://cloud.githubusercontent.com/assets/2230150/26690637/e1ba7dda-46ae-11e7-886d-67cb8eea3ea2.png)

Obviously the scale of these buildings is different, and the amount I zoomed in before starting to draw varied.  Don't think I did anything different though.

**Comment by ajwinter** [comment]
The labels are a fixed size based on zoom level, so if you have a small space, then the label will be too big.

Couple ideas:
1. The easiest is to find a good size that works consistently and stick to that. Right now the labels end up being about 100 meters wide in real world units, which is probably much too big. If we can take a stab at an average space size, then we can shrink labels accordingly.
2. We can try to fit the label size to the space size a bit more dynamically, but that will be harder.

**Comment by ljbrackney** [comment]
That makes sense.  My two examples indicate the range of scales we expect the tool to support for residential and commercial use cases.  It may be possible to use option 1 w/ label scaling appropriate for residential.  They might look a little small in some commercial spaces, but would probably be ok.

**Comment by ajwinter** [comment]
Made the labels a fair amount smaller, which seems to work for most test cases. Closing this for now, feel free to re-open!

**Comment by shorowit** [comment]
Space labels are huge when you draw at a 1ft grid resolution (common use case for residential buildings).

My suggestions:
- The space label and dimensions labels should always be the same size
- There should be a default size for the two labels (something like 10pt or 12pt font). At some point based on zooming out, the text size would decrease, but the text size would never increase beyond the default.

Screenshot of problem:
![image](https://user-images.githubusercontent.com/5861765/26901924-a68e557a-4b94-11e7-9d31-727e108c488f.png)

**Comment by ljbrackney** [comment]
+1 still

**Comment by ajwinter** [comment]
We changed this over to a fixed size, rather than adjusting by zoom level. It seems to work well in most cases. You can get some minor overlapping at some zoom levels, but that seems like an edge case. Please review and let us know your thoughts!

**Comment by ljbrackney** [comment]
This looks pretty good now, although the size labels during a drawing operation still look (temporarily) ginormous.

**Comment by shorowit** [comment]
Yes, it would be nice to have the same font size for both. I'd also like to see the font size of the space name increased, but maybe it's specific to my monitor size and screen resolution.

**Comment by shorowit** [comment]
In the latest develop, I'm seeing space names that are no longer the same font size as the space dimension labels. 

![image](https://user-images.githubusercontent.com/5861765/28883576-2afccbc2-776c-11e7-8b9e-42b169c48101.png)

**Comment by ljbrackney** [comment]
I'm seeing huge fonts again as well.

**Comment by shorowit** [comment]
One way that you can see huge fonts is to open a saved floorplan and then start drawing.

**Comment by shorowit** [comment]
The remaining issues are covered by #100.
